menu.py

```diff
@@ -8,25 +8,29 @@ def __init__(self):
         self.text_descriptions = {}
         self.functions = {}
 
+
     def add_option(self, key, description, func):
         """ Add or replaces an option to this menu.
         If another option with the same key is already present, it will be overwritten.
         :param key: the key the user should press to select this option. Can be a single character or a string
         :param description: a text description of the menu option
         :param func: the function that should be invoked when the user selects this option """
         self.text_descriptions[key] = description
```

Comment:
Will, I was able to ask Clara and get help on shorten this code. So I've updated the codes to so it will look better and shorter.